lua/lspconfig/sqls.lua

```diff
@@ -0,0 +1,34 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local root_pattern = util.root_pattern(".git", "config.yml")
+
+configs.sqls = {
+  default_config = {
+    cmd = {"sqls"};
+    filetypes = {"sql", "mysql"};
+    root_dir = function(fname)
+      return root_pattern(fname) or vim.loop.os_homedir()
```

Comment:
Should it really open in home directory? Cwd? Buf directory? (for fallback)

Reply:
changed to cwd/.git